src/query/graphite/native/aggregation_functions.go

```diff
@@ -22,11 +22,15 @@ package native
 
 import (
 	"fmt"
+	xerrors "github.com/m3db/m3/src/x/errors"
```

Comment:
nit: this should be in the second import group

Reply:
fixed